src/app/backend/events.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"time"
+)
+
+// Events response structure.
+type Events struct {
+	// Namespace.
+	Namespace string `json:"namespace"`
+
+	// Event list.
+	Events []Event `json:"events"`
+}
+
+// Single event representation.
+type Event struct {
+	// Message.
+	Message string `json:"message"`
+
+	// Source (following pattern {component host}).
+	Source string `json:"source"`
+
+	// Sub-object.
+	SubObject string `json:"object"`
+
+	// Count.
```

Comment:
This documentation can (and should) be improved :) Say what count is, not that it is count. Try explaining this field to a five years old, and write your explanation here. 
Applies for all docs in this file.